dev-debug/bpftrace/bpftrace-0.24.1.ebuild

```diff
@@ -115,18 +115,24 @@ src_configure() {
 	local mycmakeargs=(
 		# DO NOT build the internal libs as shared
 		-DBUILD_SHARED_LIBS=OFF
-		# DO dynamically link the bpftrace executable
-		-DSTATIC_LINKING:BOOL=OFF
-		-DBUILD_TESTING:BOOL=$(usex test)
+		-DBUILD_TESTING=$(usex test)
+		-DBUILD_UBSAN=$(usex sanitize)
 		# we use the pregenerated man page
-		-DENABLE_MAN:BOOL=OFF
-		-DENABLE_SYSTEMD:BOOL=$(usex systemd)
-		-DENABLE_SKB_OUTPUT:BOOL=$(usex pcap)
+		-DENABLE_MAN=OFF
+		-DENABLE_SKB_OUTPUT=$(usex pcap)
+		-DENABLE_SYSTEMD=$(usex systemd)
+		# DO dynamically link the bpftrace executable
```

Comment:
I would personally keep that right by the `BUILD_SHARED_LIBS` but it's up to you.

Reply:
Moved.